docs/UserGuide.md

```diff
@@ -178,6 +178,22 @@ Format: `add_session n/STUDENT_NAME d/DATE t/TIME k/DURATION s/SUBJECT f/FEE`
 Examples:
 * `add_session n/John Doe d/2021-01-01 t/18:00 k/120 s/Biology f/80`
 
+### Adding a recurring tuition session: `add_rec_session`
```

Comment:
Looks good, however, do remember to add this command over to the TOC and the command summary at the end of the page for consistency.